### DISCUSSION TITLE
Diskussionen

### DISCUSSION THREAD

**Discussion by GWRon** [TVTower/TVTower#313] [General]
Wir koennten die "Dev-Diskussionen" (in Github-Teams) auch gerne in die "Discussions" hier ueberfuehren  - vielleicht bringt das den einen oder anderen Githubnutzer zum mitdiskutieren?

**Comment by jukey** [comment]
Was genau ist denn Github-Teams? :)

**Reply by GWRon** [reply]
Man kann in einer "Organisation" (hier das erste "TVTower" in der Url) Teams anlegen - und dort dann wieder Diskussionen fuehren.

https://github.com/team

**Comment by nittka** [comment]
Ich bin hin- und hergerissen. Tickets haben den Charme, das man sie zumachen kann, wenn das Thema erledigt ist. Diskussionen kann man sperren, aber wenn es viele werden, kann es dennoch unübersichtlich werden. Dann müsste man sich was mit den Kategorien überlegen.
Allemeine Diskussionen könnte man hierher verlagern, aber wenn es um Implementierungsdetails geht, ist die öffentliche Diskussion vielleicht nicht der richtige Platz. Wie so häufig, sollte man genau überlegen, welcher Kanal der passende ist.
Von den aktuellen Team-Diskussionen ist die zur "Meilensteinplanung" die einzige, die ich inhaltlich hierher bewegen würde - Welche Themen sollten als nächstes angegangen werden. Dafür könnte es aber auch einen Meilenstein mit einem zugewiesenen "Themen-Ticket" geben. Das müsste nicht unbedingt eine Diskussion sein.

**Comment by engsoz** [comment]
Ich bin mir nicht sicher, ob ich hier richtig bin. Ich wollte mich einfach einmal bedanken für dieses wunderbare Spiel. Ich habe mad tv gern gespielt, aber TV-Tower liebe ich. Ich habe jetzt schon Stunden um Stunden damit verbracht und im Gegensatz zu mad tv kaum Probleme gehabt. Ich hätte vielleicht mehr kleine Probleme aufschreiben sollen, damit ich euch davon berichten kann. Ich habe noch einige in Erinnerung, aber das wird euch nicht wirklich helfen. Vielen, VIELEN DANK für diese Perle von einem Spiel! Ihr macht großartige Arbeit und ich genieße jede Minute mit diesem Spiel! Ich würde gerne ewig weiterspielen, doch bin ich schon bei 99,5% Senderimage und 94,8% Liebe. Ich werde es demnächst auch einmal auf Linux testen! Danke, daß es gratis ist, da ich im Augenblick nicht über viel Barschaft verfüge!

**Comment by engsoz** [comment]
Mir sind ein paar Fehler aufgefallen, die ich mit euch teilen wollte:
*) Wenn man oder ein Computer Spieler Prinz Charles und Lady Diana Boulevard Magazin sendet, werden die Daten nicht mehr angezeigt, wenn man mit der Maus über den Fernseher fährt. Wenn man den Spielstand neu lädt, funktioniert alles wieder.
*) Am Zuschauermonitor (Couch) sitzen manchmal Leute übereinander.
*) Bei der Neuübernahme eines Fernsehsenders durch einen Computer Spieler ändert sich die Farbe der Büros nicht sofort (Büro, Nachrichten, Boss, Archiv, Studio).
*)Am Schreibtisch bei den Nachrichten schließt sich das Nachrichtenfenster immer mal wieder anstatt die Nachrichten zu löschen beim Rechtsklick
*) Bei manchen Versteigerungen ist der Startwert höher als der Wert des Filmes (ohne Gebot)
*) Bei manchen gewonnenen Preisen ändert sich das Image nicht (Ist mir bei über 95% Senderimage aufgefallen)
*) Meldung Raum wird schon benutzt, wenn man den Aufzug benutzen will (nochmaliges Klicken behebt das Problem)

Dann möchte ich noch ein paar kleine Verbesserungen vorschlagen:
*) Einführung eines Studiomanagements (vielleicht unter dem Punkt "Kommt noch" am Computer)
*) Mehr Informationen (z.B. Serientitel und Nummer der Folge) wenn man mit dem Mauscursor über das Studio fährt
*) 100 Folgen Serie entfernen und/oder
*) Möglichkeit die Besetzung aller Folgen auf einmal durchzuführen anstatt nur 8
*) Etwas weniger Personal, das den Aufzug blockiert

So das wäre erstmal alles. Vielen Dank fürs lesen!
lg

**Reply by nittka** [reply]
Vielen Dank für die Rückmeldung.
Ein paar der Fehler sind schon bekannt, behoben und werden damit in der kommenden Version nicht mehr auftreten:
* Hover bei Boulevardmagazin weg
* Leute sitzen übereinander
* Meldung "Raum schon benutzt" beim Fahrstuhl

Für die anderen kann ich in der nächsten Zeit mal schauen, ob ich sie reproduziert und repariert bekomme.

[EDIT]: Bei den Versteigerungspreisen - war da eine Preisleveländerung involviert (Zuschauerzahl geht zurück, Level runter)? Kannst Du sagen, ob das bei Filmen war, die komplett neu in einer Auktion aufgetaucht sind oder schonmal dabei waren?

Das Image-Problem könnte an einer ungleichen Image-Verteilung pro Zielgruppe liegen. Das Gesamt-Image ist zwar 95, aber bei wichtigen Zielgruppen ist es bereits 100. Der "Preis" ist kein absoluter Imagezuwachs sondern einer pro Zielgruppe. Bei den 100ern kommt nichts mehr dazu, und 1% bei kleinen Gruppen hat wenig spürbaren Effekt.

Zu den 100er-Produktionen: (Es muss ja keine gute Idee sein, immer dieselbe Besetzung zu verwenden.) Ich vermute mal, Du kennst schon die Möglichkeit, im Supermarkt eine Besetzung auf die nächste Produktion zu übernehmen. Mein Workaround-Vorschlag wäre, die jeweils letzte Folge nur vorzubereiten und nicht abzuschließen. Dann ist sie nach Produktionsende nämlich noch da, Du kannst neue EInkaufslisten holen und die mit den Werten der übriggelassenen Planung belegen.

**Comment by engsoz** [comment]
Hi Alexander!

 

Vielen Dank für Deine Rückmeldung! Freue mich schon auf die neue Version mit den behobenen Problemen!

 

Falls Du noch etwas brauchen solltest bezüglich der Reproduktion der gemeldeten Probleme, stehe ich gerne zur Verfügung!

 

Danke für den Tipp des nicht Abschließens der letzten Folge. Trotzdem verwende ich schon seit einiger Zeit keine Serien mit 100er Folgen mehr. Ich habe jetzt alle freien Räume "übernommen" und produziere ununterbrochen Serien auf höchstem Niveau. Leider ist es schwer bis unmöglich den Überblick darüber zu behalten. Würde ich Serien mit 100 Folgen produzieren würde das ewig dauern (4 Tage pro Folge) und wäre vom Aufwand her noch anstrengender. So produziere ich Serien mit 6-12 Folgen in 6-8 Räumen.^^ Eine bessere Organisationsmöglichkeit und Überblick wären da trotzdem schon seeehr hilfreich!^^

 

Danke trotzdem für all die Mühen! Ich finde TVTower einfach genial und es macht mehr Spaß als das Orginal!

 

Alles Gute noch und viel Glück!

 

lg

Max

 

Gesendet: Dienstag, 05. Dezember 2023 um 16:53 Uhr
Von: "Alexander Nittka" ***@***.***>
An: "TVTower/TVTower" ***@***.***>
Cc: "engsoz" ***@***.***>, "Comment" ***@***.***>
Betreff: Re: [TVTower/TVTower] Diskussionen (Discussion #313)


 

Vielen Dank für die Rückmeldung.
Ein paar der Fehler sind schon bekannt, behoben und werden damit in der kommenden Version nicht mehr auftreten:


	Hover bei Boulevarmagazin weg
	Leute sitzen übereinander
	Meldung "Raum schon benutzt" beim Fahrstuhl


Für die anderen kann ich in der nächsten Zeit mal schauen, ob ich sie reproduziert und repariert bekomme.

Zu den 100er-Produktionen: (Es muss ja keine gute Idee sein, immer dieselbe Besetzung zu verwenden.) Ich vermute mal, Du kennst schon die Möglichkeit, im Supermarkt eine Besetzung auf die nächste Produktion zu übernehmen. Mein Workaround-Vorschlag wäre, die jeweils letzte Folge nur vorzubereiten und nicht abzuschließen. Dann ist sie nach Produktionsende nämlich noch da, Du kannst neue EInkaufslisten holen und die mit den Werten der übriggelassenen Planung belegen.
